ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -282,6 +288,12 @@ private Long incrementalDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot, Hive
     return lastReplId;
   }
 
+  private boolean needBootstrapAcidTablesDuringIncrementalDump() {
+    // If old replication policy is available, then it is possible some of the ACID tables might be
+    // included for bootstrap during incremental dump.
+    return (work.oldReplScope != null) || conf.getBoolVar(HiveConf.ConfVars.REPL_BOOTSTRAP_ACID_TABLES);
```

Comment:
and only if acid table dump is enabled ?

Reply:
Good catch. Will fix it and add a test for this combination.